pkg/cli/cmd/authorizer/authorizer.go

```diff
@@ -0,0 +1,10 @@
+package authorizer
+
+type AuthorizerCmd struct {
+	EvalDecision EvalDecisionCmd `cmd:"" help:"evaluate policy decision" group:"authorizer"`
+	DecisionTree DecisionTreeCmd `cmd:"" help:"get decision tree" group:"authorizer"`
+	ExecQuery    ExecQueryCmd    `cmd:"" help:"execute query" group:"authorizer"`
```

Comment:
Have we validated that we can use the output of a partial query (compile) in query, as our query semantics are not as flexible as regular OPA, which are required to use the output of partial queries?